crd-ref-docs.config.yaml

```diff
@@ -0,0 +1,10 @@
+processor:
+  # RE2 regular expressions describing Group.Version that should be excluded from the generated documentation.
+  ignoreGroupVersions:
+    - "features.opendatahub.io/v1"
+  # RE2 regular expressions describing types that should be excluded from the generated documentation.
+  ignoreTypes:
+    - "(DataScienceCluster|DSCInitialization)List$"
+render:
+  # Version of Kubernetes to use when generating links to Kubernetes API documentation.
+  kubernetesVersion: 1.22
```

Comment:
Can we rely on default here? This version is reaching end-of-life. And I would prefer not to be bothered about updating this file with k8s version :) Unless we want to have it pinned to what latest openshift is built on.

Reply:
the only issue if I remove the kubernetesVersion is that it generates a URL like this in the doc:
https://kubernetes.io/docs/reference/generated/kubernetes-api/v/#objectreference-v1-core
which is broken :(.

Reply:
@bartoszmajsak  ^

Reply:
Bummer, but then how about aligning it with what we have in `Makefile` (so setting it for now to 1.24 as that matches our envtest version at least)?

Reply:
done. It is now 1.24